benchmarks/nemo/nemolite2d/manual_versions/regent/model_init.rg

```diff
@@ -359,12 +360,20 @@ do
     init_centre_launcher(centre_region[int2d({0,0})])
 
     init_west(west_region[int2d({0,0})]) 
-
-    init_east(east_region[int2d({0,0})])
+    
+    if( jphgr_msh == 2 or jphgr_msh == 3) then
+      init_east(east_region[int2d({0,0})], TIDAL_MASK)
+    else
+      init_east(east_region[int2d({0,0})], SOLID_BOUNDARY)
+    end
 
     init_north(north_region[int2d({0,0})])
```

Comment:
Note that I changed the order of boundary order inits  to west north east south. This affects the value of the NE corner, so it may be better to change it here as well.

Reply:
Hmm if the value of the NE corner has changed I need to do a bit more work than reordering, as the east_region and north_region declarations are defined to be non-overlapping, so ordering is irrelevant (and they can execute in parallel). I'll have another look in a bit and check this matches still.

Reply:
It's only the value of the T-mask in the *boundary* region, not inside the simulation domain (in case that helps you).

Reply:
I think not - it shouldn't be a big change either way, the simulation domain is all initialised by `init_centre` (or the launcher since that is also done in parallel).

Reply:
I had forgotten about this - I'll try to look at this soon and fix it so this can be merged.

Reply:
Having just reviewed Sergi's new PR, I see this one is still open. Do you (eventually) want to proceed with it @LonelyCat124 ?

Reply:
I have forgotten what I needed to do - I think it would be good to do in principle but I don't remember what options we needed to add.